.vscode/settings.json

```diff
@@ -129,5 +129,23 @@
         "yaml.schemas": {
                 "${workspaceFolder}/validation/module_schema.yaml": "${workspaceFolder}/src/modules/*/module.yaml"
         },
-        "ros.distro": "humble"
+        "ros.distro": "humble",
+        "python.autoComplete.extraPaths": [
+                "/home/roque/discower_ws/build/px4_mpc",
+                "/home/roque/discower_ws/install/px4_mpc/lib/python3.12/site-packages",
+                "/home/roque/discower_ws/build/px4_offboard",
+                "/home/roque/discower_ws/install/px4_offboard/lib/python3.12/site-packages",
+                "/home/roque/discower_ws/install/px4_msgs/lib/python3.12/site-packages",
+                "/home/roque/discower_ws/install/mpc_msgs/lib/python3.12/site-packages",
+                "/opt/ros/jazzy/lib/python3.12/site-packages"
+        ],
+        "python.analysis.extraPaths": [
+                "/home/roque/discower_ws/build/px4_mpc",
+                "/home/roque/discower_ws/install/px4_mpc/lib/python3.12/site-packages",
+                "/home/roque/discower_ws/build/px4_offboard",
+                "/home/roque/discower_ws/install/px4_offboard/lib/python3.12/site-packages",
```

Comment:
Local configurations propagating into the repo I presume?

Reply:
A classic... We have .vscode/settings in the tracked files, how do you solve this locally? Should it be tracked?